js/data/sykepengerVarsel/sykepengerVarselSagas.js

```diff
@@ -0,0 +1,45 @@
+import {
+    call, fork, put, select, takeEvery, all,
+} from 'redux-saga/effects';
+import { log } from '@navikt/digisyfo-npm';
+import { get, hentApiUrl } from '../gateway-api/index';
+import * as actions from './sykepengerVarselActions';
+import logger from '../../logging';
+import { MANGLER_OIDC_TOKEN } from '../../enums/exceptionMessages';
+import {
+    HENT_SYKEPENGERVARSEL_FORESPURT,
+} from './sykepengerVarselActionTyper';
+import { skalHenteSykepengerVarsel } from './sykepengerVarselSelectors';
+
+export function* hentSykepengerVarsel() {
+    yield put(actions.henterSykepengerVarsel());
+    try {
+        const data = yield call(get, `${hentApiUrl()}/syfosyketilfelle/39ukersvarsel`);
+        yield put(actions.sykepengerVarselHentet(data));
+    } catch (e) {
+        log(e);
+        if (e.message === MANGLER_OIDC_TOKEN) {
+            yield put(actions.henterSykepengerVarsel());
+        } else {
+            logger.error(`Kunne ikke hente status på sykepengervarsel fra syfosoknad. URL: ${window.location.href} - ${e.message}`);
```

Comment:
hva er forskjell på `log` og `logger`?

Reply:
Logger går mot frontendlogger, mens log er hentet fra digisyfo-npm. Beholder begge for å følge kodestilen for sagaer i dette prosjektet.